hugegraph-llm/llm_api/requirements.txt

```diff
@@ -0,0 +1,6 @@
+torch==2.1.2
+transformers==4.39.3
+fastapi==0.110.1
+accelerate==0.29.2
+chardet==5.2.0
```

Comment:
can we remove [chardet](https://github.com/chardet/chardet), apache project cannot introduce dependencies under the LGPL license.